go/vt/vttablet/tabletserver/vstreamer/local_vschema_test.go

```diff
@@ -63,21 +63,12 @@ func TestFindColVindex(t *testing.T) {
 							Columns: []string{"id"},
 						}},
 					},
-					"nogoodvindex1": {
+					"nogoodvindex": {
 						ColumnVindexes: []*vschemapb.ColumnVindex{{
 							Name:    "lookup_unique",
 							Columns: []string{"id"},
 						}},
 					},
-					"nogoodvindex2": {
```

Comment:
It seems that removing this leads to a loss of coverage, since we are now testing only with lookup_unique and not lookup.

Reply:
This test wasn't really covering that unreachable code path, which is why I added the comment. So, it wasn't testing anything that other tests already covered. There is an upcoming PR where I move this code to the vindexes package. I cover that code path there, because I have specific test-only videxes for covering it.